pkg/splunk/enterprise/indexercluster_test.go

```diff
@@ -169,7 +220,10 @@ func indexerClusterPodManagerReplicasTester(t *testing.T, method string, mockHan
 	replicas int32, desiredReplicas int32, wantPhase splcommon.Phase,
 	wantCalls map[string][]spltest.MockFuncCall, wantError error) {
 
-	mgr, mockSplunkClient := getIndexerClusterPodManagerAndClient(method, mockHandlers, replicas)
+	mockSplunkClient := &spltest.MockHTTPClient{}
```

Comment:
Any reason we moved **mockSplunkClient** creation from **getIndexerClusterPodManagerAndClient** to here? Was it just a cosmetic change to avoid returning a **mockSplunkClient** return value from the function **getIndexerClusterPodManagerAndClient**?

Reply:
I did this because in many test cases I am not using `mockSplunkClient` like I am using in this function( `mockSplunkClient.CheckRequests(t, method)`). Hence I had to take it out of `getIndexerClusterPodManager` or else it would have given a compilation error of `variable defined but not used`.